packages/@aws-cdk/aws-eks/README.md

```diff
@@ -375,6 +375,23 @@ cluster.connectAutoScalingGroupCapacity(asg);
 
 This will add the necessary user-data and configure all connections, roles, and tags needed for the instances in the auto-scaling group to properly join the cluster.
 
+`addAutoScalingGroupCapacity` and `connectAutoScalingGroupCapacity` add spot interrupt handler by default, to disable default behavior, set `addSpotInterruptHandler` to false.
+
+```ts
+cluster.addAutoScalingGroupCapacity("spot", {
+  instanceType: new ec2.InstanceType("t3.large"),
+  minCapacity: 2,
+  addSpotInterruptHandler: false,
+});
+```
+
+```ts
+const asg = new ec2.AutoScalingGroup(...)
+cluster.connectAutoScalingGroupCapacity(asg, {
+  addSpotInterruptHandler: false
+});
+```
+
```

Comment:
Move this to here: https://github.com/aws/aws-cdk/tree/master/packages/%40aws-cdk/aws-eks#spot-instances

It's also a little verbose. No need for the code examples since we already show them, just mention that `addSpotInterruptHandler` can be used to disable installation of the spot termination handler.